src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -975,52 +976,52 @@ private boolean addEntry(final Map<String, List<MapEntry>> entryMap, final Strin
     }
 
     /**
-     * Load aliases Search for all nodes inheriting the sling:alias
-     * property
+     * Load aliases - Search for all nodes (except under /jcr:system) below
+     * configured alias locations having the sling:alias property
      */
     private Map<String, Map<String, String>> loadAliases(final ResourceResolver resolver) {
         final Map<String, Map<String, String>> map = new ConcurrentHashMap<>();
-        final String queryString = updateAliasQuery();
+        final String queryString = generateAliasQuery();
+
         log.debug("start alias query: {}", queryString);
+        long queryStart = System.nanoTime();
         final Iterator<Resource> i = resolver.findResources(queryString, "sql");
-        log.debug("end alias query");
+        long queryElapsed = System.nanoTime() - queryStart;
```

Comment:
I am not sure how relevant this information is, because the query itself is executed lazily (IIRC it just loads 20 hits on execution), and just fetching the additional result rows gets costly.

I would drop this information and just collect the "processElapsed" value you capture below.

Reply:
Does it hurt? Reminder: this is for people looking at DEBUG logs.